app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryScreenModel.kt

```diff
@@ -300,10 +301,18 @@ class LibraryScreenModel(
                     val item2Score = trackerScores[i2.libraryManga.id] ?: defaultTrackerScoreSortValue
                     item1Score.compareTo(item2Score)
                 }
+                LibrarySort.Type.Random -> {
+                    error("A comparator should not be requested for the random sort style. Instead, intercept this " +
+                        "case and call .shuffle()")
```

Comment:
As this is a local function it's fine to just have
```suggestion
                    error("Why Are We Still Here? Just To Suffer?")
```

Reply:
lmao sounds good